rs/rosetta-api/icp/tests/system_tests/test_cases/neuron_management.rs

```diff
@@ -30,6 +35,7 @@ use tokio::runtime::Runtime;
 
 lazy_static! {
     pub static ref TEST_IDENTITY: Arc<BasicIdentity> = Arc::new(test_identity());
+    pub static ref NUM_TEST_CASES: u32 = 1;
```

Comment:
nit: Could this be a plain `const` outside of the `lazy_static`?

Reply:
Yes, but is there a benefit to that?

Reply:
Okay, thanks for pointing that out. I can create a ticket to change the pattern for all tests.